app/models/actor.rb

```diff
@@ -12,27 +12,21 @@
 #
 class Actor < ApplicationRecord
   def characters
-    my_id = self.id
+    key = self.id
```

Comment:
🤔 I feel like `foreign_key` or `key` maybe connects more to a database perspective of things, while `my_id` is helpful at the Ruby instance level method type of way. It's not clear to me if one is more clear to students than the other.

Reply:
Yeah. I wanted to try `key` in this second refactoring project since we're ramping into `:foreign_key => ...`, the more database-y view of things.

Reply:
In principle, I would feel better about something like:

```
criteria = self.id
foreign_key = :actor_id
the_many = Character.where({ foreign_key => criteria })
```

but that seems a bit verbose.

Reply:
> Yeah. I wanted to try `key` in this second refactoring project since we're ramping into `:foreign_key => ...`, the more database-y view of things. And `key` rather than `foreign_key` since in some places, like on this line, we're assigning an ID and it might be confusing since that's a primary key (and not yet obvious that the intention is to _use_ it as a foreign key).

That makes sense to me. I still think it should mostly be clear.



> In principle, I would feel better about something like:
> 
> ```ruby
> criteria = self.id
> foreign_key = :actor_id
> the_many = Character.where({ foreign_key => criteria })
> ```
> 
> but that seems a bit verbose.

I think would also introduce confusion in using a variable as the symbol argument.